conduct/code-of-conduct.md

```diff
@@ -1,17 +1,21 @@
 # Code of Conduct
 
-R-Ladies is dedicated to providing a harassment-free experience for everyone. We do not tolerate harassment of participants in any form.
+`<EVENT>` is dedicated to providing a harassment-free experience for participants at our event(s). `<EVENT>` is a working conference intended for professional networking and collaboration in the R community. They exist to encourage the open exchange of ideas and expression and require an environment that recognizes the inherent worth of every person and group. 
 
-This code of conduct applies to all R-Ladies spaces, including meetups, Twitter, Slack, mailing lists, both online and offline. Anyone who violates this code of conduct may be sanctioned or expelled from these spaces at the discretion of the Global Leadership Team.
+A **Code of Conduct** is useful to define accepted and acceptable behaviors and to promote high standards of professional practice. It also provides a benchmark for self evaluation and acts as a vehicle for better identity of the community and organization.
```

Comment:
I wonder if it's worth adding a link to a FAQ here? Maybe `If you have general questions about codes of conduct, a good place to start is this [code of conduct FAQ](https://www.ashedryden.com/blog/codes-of-conduct-101-faq#cocfaq)`